StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedMandateContainerView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  EmbeddedMandateContainerView.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 10/3/24.
+//
+
+import Foundation
+import UIKit
+
+/// A simple container that holds a mandate text view for the embedded payment element
+class EmbeddedMandateContainerView: UIView {
```

Comment:
ooc any reason to not put `SimpleMandateTextView` directly in `EmbeddedPaymentElement`?  Maybe we can add the reason to the docstring

Reply:
I think it's to encapsulate the constraint code, b/c we're putting it in a stackview and have to do some things to make it look right.

I'm not convinced that it should be in the stackview though.  Like, should it respect the stackview's `spacing`?

Reply:
Yeah it pretty much just encapsulates the constraint code so we get proper padding. I updated the doc comment a bit.

If we directly add the mandate view to the `EmbeddedPaymentMethodsView`, when we hide it we need to do some manual calculation to get the height of the `EmbeddedPaymentMethodsView` correct (at least from what I could see in my testing). But when it's inside the stack view, the stack view auto resizes itself when we hide the mandate which makes it easy to get the height of the EmbeddedPaymentMethodsView.

re: Spacing good callout, I set custom spacing before mandate to be 0.

Reply:
I think what you're describing is the fact that a stackview will automatically remove a hidden arranged subview / make its height zero. That's a nice benefit of it being in a stackview for sure.